### DISCUSSION TITLE
Web push notification

### DISCUSSION THREAD

**Discussion by hkalbasi** [community/community#9983] [Other Feature Feedback, Questions, & Ideas]
I don't use mobile very often, but I want to see a notification for notifications. Web has a mature push notification api which can deliver them even when there is no open tab. Can github use it?

**Comment by copygirl** [comment]
I was at first confused as to why this is not already a feature, and it took a while to get here. I would like to see web push notifications (not sure what they're called) from GitHub in my browser. I want to be able to get these notifications for every notification that also shows up on `github.com/notifications`, so I can then see them pop up on my desktop, as I am already able to do with other web services.

I know this would be possible by turning on email notifications, but I'd rather not have those pile up in my inbox, and I'm already using GitHub's notification page and quite happy with it. I just want to be informed as they come in, and decide when to start responding to them, potentially right away.

**Reply by mikecharles** [reply]
Yeah I do not understand why this isn't already a feature

**Comment by WoodyWoodsta** [comment]
Seems to me like a reasonable ask even a year later :)

I imagine that most are either using web + mobile notifications or email notifications. I'd quite like to switch off email notifications and use just web and mobile, but for that I would need push notifications for the web!

**Comment by xander-marjoram** [comment]
I'm pretty sure Gitlab has this feature and it saved me quite a lot of time even for a single merge request. I can't imagine how much time this would save me if it were on all of my GitHub PRs.

**Comment by jamieomaguire** [comment]
Yep, this would save me heaps of time. I'm surprised it isn't already a feature to be honest.
